examples/reference/widgets/Tabulator.ipynb

```diff
@@ -3,7 +3,9 @@
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {},
+   "metadata": {
+    "tags": []
```

Comment:
Uff, these are annoying to look at. Did you run this in Jupyterlab?

Reply:
Oh yes indeed, that's not nice. I will clear that. I'm not sure how it got there, I opened it both in JLab and VSCode.